serverless/plugin/step_functions.json

```diff
@@ -1221,6 +1261,6 @@
         "required": ["End"]
       }
     ],
-    "required": ["Type", "Iterator"]
+    "required": ["Type", "ItemProcessor"]
```

Comment:
Let's make two different schemas to do this.

Reply:
If I understand it correctly, we have to create two different objects like
```
Aws_StateMachine_States_Map_Iterator: {}
Aws_StateMachine_States_Map_ItemProcessor: {}
``` 
and use oneOf in `Aws_StateMachine_Definition` ?

Reply:
I hope this is what you meant by two different schemas. Let me know if I understood it wrong. I'm happy to learn and change the code.

Reply:
Yes, this is what I was looking for. I just wanna test this once